<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="3ed03c57-4d33-471e-97d2-c13c7b884ade" name="jciechanowicz@AS02W1018 2012-11-06 16:35:21" runUser="ALPHAMERIC\jciechanowicz" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="42885817-39b2-4c60-8baf-effd82c36c3a">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="jciechanowicz_AS02W1018 2012-11-06 16_35_21">
      <DeploymentItem filename="C:\Users\jciechanowicz\Documents\GitHub\Chains\Chains\bin\Debug\Chains.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2012-11-06T16:35:21.8938223+00:00" queuing="2012-11-06T16:35:25.1678223+00:00" start="2012-11-06T16:35:25.4248223+00:00" finish="2012-11-06T16:35:38.6188223+00:00" />
  <ResultSummary outcome="Error">
    <Counters total="1" executed="1" error="0" failed="0" timeout="0" aborted="1" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
    <RunInfos>
      <RunInfo computerName="AS02W1018" outcome="Warning" timestamp="2012-11-06T16:35:22.6018223+00:00">
        <Text>Deployment is currently disabled. The deployment item 'Chains.dll' on test 'ChainsTest.ChainsTests.TestAsyncTypeError' was not deployed.</Text>
      </RunInfo>
      <RunInfo computerName="AS02W1018" outcome="Error" timestamp="2012-11-06T16:35:38.6158223+00:00">
        <Text>Test host process exited unexpectedly.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="TestAsyncTypeError" storage="c:\users\jciechanowicz\documents\github\chains\chainstest\bin\debug\chainstest.dll" id="a84f3122-8fdf-d178-f2df-bd9079d6ec22">
      <DeploymentItems>
        <DeploymentItem filename="Chains.dll" />
      </DeploymentItems>
      <Execution id="5e7b8125-93c2-4783-9732-895403ec7ace" />
      <TestMethod codeBase="C:/Users/jciechanowicz/Documents/GitHub/Chains/ChainsTest/bin/Debug/ChainsTest.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="ChainsTest.ChainsTests, ChainsTest, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="TestAsyncTypeError" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="a84f3122-8fdf-d178-f2df-bd9079d6ec22" executionId="5e7b8125-93c2-4783-9732-895403ec7ace" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="5e7b8125-93c2-4783-9732-895403ec7ace" testId="a84f3122-8fdf-d178-f2df-bd9079d6ec22" testName="TestAsyncTypeError" computerName="AS02W1018" duration="00:00:12.9450000" startTime="2012-11-06T16:35:25.6738223+00:00" endTime="2012-11-06T16:35:38.6188223+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Aborted" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="5e7b8125-93c2-4783-9732-895403ec7ace">
    </UnitTestResult>
  </Results>
</TestRun>